assets/src/blocks/editorIndex.js

```diff
@@ -8,4 +9,5 @@ wp.domReady(() => {
   // Beta blocks
   registerActionsList();
   registerPostsListBlock();
+  registerHappypointBlock();
```

Comment:
The Happypoint isn't a beta block 🤔 Does this mean the Actions List and Posts List blocks also show when the beta blocks aren't enabled?

Reply:
For the other blocks we moved so far, we used [webpack.config.js](https://github.com/greenpeace/planet4-master-theme/blob/main/webpack.config.js#L36-L45)

Reply:
In that case we'll need to add a script to register the block

Reply:
We can actually use this file I think, just not add the Happypoint under `Beta blocks`, see this [conversation](https://github.com/greenpeace/planet4-master-theme/pull/2195#discussion_r1453210087) I've had with Osong about the Submenu block which was similar 🙂